Docs/Configuration.tex

```diff
@@ -1329,19 +1329,18 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Disable single user mode.
 
-  This is a security option allowing one to restrict single user mode usage
+  This is a security option that restricts activating single user mode
```

Comment:
restricts *the activation of*...

Reply:
Think current is equally valid but will change